llvm/include/llvm/Transforms/Vectorize/LoopVectorizationLegality.h

```diff
@@ -404,6 +424,22 @@ class LoopVectorizationLegality {
 
   DominatorTree *getDominatorTree() const { return DT; }
 
+  /// Returns all exiting blocks with a countable exit, i.e. the
+  /// exit-not-taken count is known exactly at compile time.
+  const SmallVector<BasicBlock *, 4> &getCountableExitingBlocks() const {
```

Comment:
I think it would be better to use `SmallVectorImpl<BasicBlock *>` instead of tying the result type to the default parameter.